pahfit/base.py

```diff
@@ -479,9 +498,25 @@ def save(self, obs_fit, filename, outform):
                         component.fwhm.bounds[0],
                         component.fwhm.bounds[1],
                         component.fwhm.fixed,
+                        strength,
+                        strength_unc,
                     ]
                 )
             elif comp_type == "Gaussian1D":
+
+                # Calculate line strength.
+                # TODO: Use input units to determine conversion and line strength units.
+                fwhm = 2 * component.stddev.value * np.sqrt(2 * np.log(2))
+                strength = ((np.sqrt(np.pi / np.log(2)) / np.pi) * const.c.to('micron/s').value / 2) * (component.amplitude.value * fwhm / component.mean.value**2) * 1e-26
+                # Estimate profile-weighted uncertainty.
+                if strength != 0.:
+                    nu = const.c.to('micron/s').value / x.value
+                    dnu = np.diff(nu, prepend=nu[1])
+                    uwv = np.where(component(x.value) / np.nanmax(component(x.value)) > 0.1)[0]
+                    strength_unc = np.sqrt(np.sum(np.take(yerr, uwv)**2 * np.take(dnu, uwv)**2)) * 1e-26
```

Comment:
same comment as above about the method (vs IDL pahfit).

Reply:
Yeah we want to leave "alternative estimates" fully out, even as a temporary work-around.  Of course this puts the burden on us to solve the "missing LM parameter uncertainties" problem, but that is absolutely crucial, so keeping that burden intact is fully appropriate !

Reply:
No problem. I can remove the profile weighted uncetainties out, and re-introduce them if needed in a future PR.